packages/devextreme/testing/tests/DevExpress.ui.widgets/chat.tests.js

```diff
@@ -371,6 +370,65 @@ QUnit.module('renderMessage', moduleConfig, () => {
 
         assert.strictEqual($(lastBubble).text(), text);
     });
+
+    QUnit.test('Message list should not run invalidate if 1 new message has been added to items', function(assert) {
```

Comment:
I suggest u creating a module "items change performance" and move common part of the tests there.
Main idea is to show that these tests are for specific purpose and will be mostly changed in the future

Reply:
[35f6c6878669aa8f23833235d2f12a4dca2d07ba]